### PR TITLE
Updated to axum 0.7.1, dropped http-body dependency

### PR DESCRIPTION
**Pull request by spigaz**
Hi,

I did my best to update it to support axum 0.7.1.

I also updated the example to axum 0.7.1 and workers 0.0.18.

I executed the tests on chrome, firefox and safari, and tested the example.

I also added .cargo/config.toml
[build]
target = "wasm32-unknown-unknown"

So that it works fine with cargo build and cargo check.

Cheers! Thanks!

### CODE REVIEW

**Comment by spigaz** [comment]
Sorry for two things:

I removed the previous pull request because I didn't knew it automatically updated with new commits on the branch.

I committed by mistake a flag that requires nightly, but its all fixed now, it should be working fine now for stable.

Thanks!

**Comment by logankeenan** [comment]
@spigaz - Thanks for the contribution. 

Looks like I'm not running the test CI job on PRs. I just updated that. I'll take a look at the other failing issue too.

File: example/wrangler.toml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-name = "axum-cloudflare-adapter-example"
+workers_dev = true
+
 main = "build/worker/shim.mjs"
-compatibility_date = "2022-01-20"
```
Comment (logankeenan) [inline example/wrangler.toml:3]: Looks like we need this back, was there a reason it needed to be removed?

Comment (spigaz) [reply example/wrangler.toml:3]: But yes, for some unknown reason to me its breaking the build, so I placed it back in.

Sorry for that.

**Comment by spigaz** [comment]
Well, I had to do my best to give back, as your lib helped me a lot to get started.

Sorry for the issues on the wrangler.toml, I guessing the file I used as reference wasn't okay, although updated to 0.0.18.

Thanks!

**Pull request merged**